python/lsst/ctrl/events/eventsLib.i

```diff
@@ -61,15 +61,24 @@ Access to the lsst::ctrl::events classes
 
 %shared_ptr(lsst::ctrl::events::EventFormatter)
 
+
 %import "lsst/daf/base/baseLib.i"
 %import "lsst/pex/logging/loggingLib.i"
-%import "lsst/pex/policy/policyLib.i"
 
+%typemap(out) std::vector<std::string > {
```

Comment:
Why did you find a typemap necessary?  I would have thought SWIG could do this itself if you do:

    %template(VectorString) std::vector<std::string>;